### PR TITLE
Add personalized text icon

### PR DESCRIPTION
**Pull request by talbshoam**
### WHY are these changes introduced?

The email team is currently working on adding the magic-ui component in order to allow merchants to generate subject lines using the power of AI. Since the magic icon (green circle) is very similar to the current icon we use for personalized text (purple circle), we decided to create a new icon for personalized text.

Screenshot 2023-04-11 at 9 45 24 PM

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

File: polaris-icons/icons/PersonalizedTextMajor.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kyledurand) [inline polaris-icons/icons/PersonalizedTextMajor.svg]: Can you run this through an optimizer like https://jakearchibald.github.io/svgomg/ ?

File: .changeset/moody-timers-build.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': minor
+---
+
+add new personalized text icon
```
Comment (kyledurand) [inline .changeset/moody-timers-build.md]:
```suggestion
Added PersonalizedTextMajor icon
```

File: polaris-icons/icons/PersonalizedTextMajor.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - Personalized text
+  - Personalization
+  - Customer attributes
+  - Customer
+  - Dynamic content
+authors:
+  - Natalie Hercun
+version: 1
+date_added: 2023-04-11
+date_modified: YYYY-MM-DD
```
Comment (kyledurand) [inline polaris-icons/icons/PersonalizedTextMajor.yml]:
```suggestion
date_modified: 2023-04-11
```

File: polaris-icons/icons/PersonalizedTextMajor.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name: Personalized text
+set: major
+description: Concept | Used to represent personalized text features.
```
Comment (kyledurand) [inline polaris-icons/icons/PersonalizedTextMajor.yml]:
```suggestion
description: Used to represent personalized text features.
```

File: polaris-icons/icons/PersonalizedTextMajor.svg (hunk 1 above)
Comment (talbshoam) [reply polaris-icons/icons/PersonalizedTextMajor.svg]: done

File: polaris-icons/icons/PersonalizedTextMajor.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><g fill="#616A75" clip-path="url(#a)"><path d="M2.707 9.707A.996.996 0 0 0 3 9V5a1 1 0 0 1 1-1 1 1 0 0 0 0-2C2.346 2 1 3.346 1 5v3.586l-.707.707a.999.999 0 0 0 0 1.414l.707.707V15c0 1.654 1.346 3 3 3a1 1 0 0 0 0-2 1 1 0 0 1-1-1v-4a.997.997 0 0 0-.293-.707L2.414 10l.293-.293ZM19.924 9.617a1 1 0 0 0-.217-.324L19 8.586V5c0-1.654-1.346-3-3-3a1 1 0 1 0 0 2 1 1 0 0 1 1 1v4a.996.996 0 0 0 .293.707l.293.293-.293.293A.996.996 0 0 0 17 11v4a1 1 0 0 1-1 1 1 1 0 0 0 0 2c1.654 0 3-1.346 3-3v-3.586l.707-.707a1 1 0 0 0 .217-1.09ZM10 9a2.5 2.5 0 1 0 0-5 2.5 2.5 0 0 0 0 5Z"/><path d="M5 13.375c0 1.036.84 1.875 1.875 1.875h6.25c1.036 0 1.875-.84 1.875-1.875 0-1.036-.937-3.125-5-3.125-4.062 0-5 2.09-5 3.125Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs></svg>
```
Comment (kyledurand) [inline polaris-icons/icons/PersonalizedTextMajor.svg]: I think this is still going to fail our icon checks. 

The fill of `#fff` isn't allowed, only fills of `#5C5F62` are allowed. I think the fill="none" can be removed.

width and height attrs should be removed

I don't think the clip path will pass either. Make sure the icon was exported as a flat icon without extra layers.

**Review by kyledurand — COMMENTED** [review]
Looking pretty good but I think the tests will fail because there's some stuff that can be removed from the markup. Let me know if you need a hand with it

File: polaris-icons/icons/PersonalizedTextMajor.svg (hunk 5 above)
Comment (nataliehercun) [reply polaris-icons/icons/PersonalizedTextMajor.svg]: hey @kyledurand ! 

Trying to make sure I export things correctely [from figma](https://www.figma.com/file/0v0kAm6H8Do13rJj4SpIpp/Email-text-assistant?node-id=1834%3A114214&t=4JVQRIgancMRgNny-1). I removed the background fill, but not sure what I need to do to optimize for the following: 
- width and height attrs should be removed
- I don't think the clip path will pass either. Make sure the icon was exported as a flat icon without extra layers.

**Review by kyledurand — APPROVED** [review]

**Pull request merged**